# HG changeset patch
# User Rik <rik@octave.org>
# Date 1517518217 28800
#      Thu Feb 01 12:50:17 2018 -0800
# Node ID 03546fb2490ffbcdc73515ef56a9433d4a816bcb
# Parent  4a9793818fcaf3f51e566ceb12695bad92f66e4d
file-editor-tab.cc: Fix compilation warning about unused variable.

* file-editor-tab.cc (check_valid_codec): Don't declare a variable to capture
the unused output of QMessageBox::Critical.

diff --git a/libgui/src/m-editor/file-editor-tab.cc b/libgui/src/m-editor/file-editor-tab.cc
--- a/libgui/src/m-editor/file-editor-tab.cc
+++ b/libgui/src/m-editor/file-editor-tab.cc
@@ -2289,17 +2289,17 @@ file_editor_tab::check_valid_identifier 
   return false;
 }
 
 bool
 file_editor_tab::check_valid_codec (QTextCodec *codec)
 {
   if (! codec)
     {
-      int ans = QMessageBox::critical (nullptr,
+      QMessageBox::critical (nullptr,
             tr ("Octave Editor"),
             tr ("The current codec %1\n"
                 "can not be applied.\n\n"
                 "Please select another one or cancel saving!").arg (_encoding));
 
       return true;
     }
 
